.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ executors:
     docker:
       # NOTE: To upgrade the Go version, first push the upgrade to the cross-builder Dockerfile
       # in the edge repo, then update the version here to match.
-      - image: quay.io/influxdb/cross-builder:go1.17.8-d52942c853d412a5bbcc92f5d78c7c5126c0730d
+      - image: quay.io/influxdb/cross-builder:go1.17.13-c1ba527d2d7854f003481f0a0aebf55524d4d986
```

Comment:
why 1.17.13 instead of 1.18.5?

Reply:
I'm not actually sure. I just kept it in the `1.17.X` line. I'll try rebuilding it with `1.18.5` instead.

Reply:
All right, it looks like everything is working!